install.bat

```diff
@@ -617,6 +624,66 @@ local function restore_config_files()
   vars.CONFBACKUPDIR = nil
 end
 
+-- Find GCC based toolchain
+local find_gcc_suite = function()
+
+    -- read output os-command
+    local read_output = function(cmd)
+        local f = io.popen("type NUL && " .. cmd .. ' 2>NUL')
+        if not f then return nil, "failed to open command: " .. tostring(cmd) end
+        local lines = {}
+        while true do
+            local l = f:read()
+            if not l then
+                f:close()
+                return lines
+            end
+            table.insert(lines, l)
+        end
+    end
+    
+    -- returns: full filename, path, filename
+    local find_file = function(mask, path)
+        local cmd
+        if path then
+            cmd = 'where.exe /R "' .. path .. '" ' .. mask
+        else
+            cmd = 'where.exe ' .. mask

```

Comment:
I assume `where.exe` is a standard Windows command? (it's not included in the LuaRocks repo or this PR)

Reply:
yes, from Windows Server 2003 onwards (so not on Win XP)

Reply:
its similar to unix `which`